rackcity/views/it_model_views.py

```diff
@@ -47,6 +47,27 @@ def model_add(request):
     )
 
 
+@api_view(['GET'])
+@permission_classes([IsAuthenticated])
+def model_detail(request, id):
+    """
+    Retrieve a single model.
+    """
+    try:
+        model = ITModel.objects.get(id=id)
+        model_serializer = ITModelSerializer(model)
+        instances = ITInstance.objects.filter(model=id)
+        instances_serializer = ITInstanceSerializer(instances, many=True)
```

Comment:
I didn't use the Recursive one here because we don't need the model info repeated, right?

Reply:
agreed